### PR TITLE
Merge branch 'efficientnet-implimentation' into main

### PR DESCRIPTION
**Pull request by dkblackley**

### CODE REVIEW

**Comment by dkblackley** [comment]
Efficient net successfully trains on training data and predicts on validation set
